tests/integration/ledger/mocks/tendermint.go

```diff
@@ -0,0 +1,37 @@
+package mocks
+
+import (
+	"context"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+	rpcclient "github.com/tendermint/tendermint/rpc/client"
+	rpcclientmock "github.com/tendermint/tendermint/rpc/client/mock"
+	coretypes "github.com/tendermint/tendermint/rpc/core/types"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+type MockTendermintRPC struct {
```

Comment:
q why not use the `testify/mock` package?

Reply:
Got 'as is' from cosmos-sdk

Reply:
not sure what that comment means

Reply:
@fedekunze I mean that I got this whole struct from cosmos-sdk repo, because I found cli execution tests example that uses It. As I mentioned before It is not accessible from dependencies as imported mock for now but It can be replaced once we will upgrade to proper sdk version. So I just copied and pasted this mock without changes and it can be replaced with import in future.

Should I replace it with one generated by mockery?